edgehog-device-runtime-containers/src/store/container.rs

```diff
@@ -435,12 +467,49 @@ fn create_container(
     network_mode: String,
     hostname: String,
     restart_policy: String,
+    cpu_period: i64,
+    cpu_quota: i64,
+    cpu_realtime_period: i64,
+    cpu_realtime_runtime: i64,
+    memory: i64,
+    memory_reservetion: i64,
```

Comment:
typo in `reserveretion` -> `reservation`, change all the occurrences.
```suggestion
    memory_reservation: i64,
```